packages/aquedux-server/src/constants/types.js

```diff
@@ -3,6 +3,20 @@
 import type { Store as ReduxStore, Dispatch as ReduxDispatch } from 'redux'
 import type { AqueduxActionType } from './actionTypes'
 
+export type AqueduxConfiguration = {
+  // The action count after which the redis queue must compact itself to 1 action.
+  queueLimit?: number,
+  // The action types that should be routed to redis.
+  statefullTypes?: Array<string>,
+  routePrefix?: string,
+  secret?: string,
+  redisHost?: string,
+  redisPort?: string,
+  onConnection?: any => void,
+  onClose?: any => void,
+  doFragmentSnapshot?: any => void
+}
+
```

Comment:
If you export the type from packages/aquedux-server/src/managers/configManager.js:28 then this definition should be removed.